index.src.html

```diff
@@ -388,7 +388,7 @@ <h4 id="types-member">
       which a user agent implements (prerendered pages, script caches, shader
       caches, etc.).
 
-      Implementation details are in [[#clear-cache]].
+      Implementation details are in <a>clear cache</a>.
```

Comment:
This was fine before: you're pointing to the section where implementation details are found.